src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2037,6 +2039,7 @@ because xbuild doesn't support framework reference assemblies.
     Condition="'$(AndroidEnableMultiDex)' == 'True' And '$(AndroidCustomMainDexListFile)' == ''"
     ToolPath="$(MainDexClassesToolPath)"
     ToolExe="$(MainDexClassesToolExe)"
+    ProguardHome="$(_MonoAndroidToolsDirectory)proguard"
```

Comment:
$(ProguardToolPath) already existed and I avoided possible different use of the variable. Though as long as I reviewed it seems to be alright to use it.